.github/workflows/haskell.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       # ------------------------------------------------------------------------
       PACKCHECK_LOCAL_PATH: "./packcheck.sh"
       PACKCHECK_GITHUB_URL: "https://raw.githubusercontent.com/harendra-kumar/packcheck"
-      PACKCHECK_GITHUB_COMMIT: "563702bd02c41343dcd3dfcfef0845ca428a9240"
+      PACKCHECK_GITHUB_COMMIT: "v0.5.1"
```

Comment:
v0.5.1 is too old (more than a year old) and it is older than the commit you replaced.